parts/linux/cloud-init/nodecustomdata.yml

```diff
@@ -350,6 +350,22 @@ write_files:
     net.bridge.bridge-nf-call-iptables = 1
     #EOF
 
+{{- if IsKubenet }}
+- path: /usr/local/bin/configure_masq_rules.sh
+  permissions: "0755"
+  owner: root
+  content: |
+    #!/bin/bash
+    set -x
+    # we are currently enabling ipmasq for kubenet cni with containerd which has the unfortunate
+    # side effect of SNATing all outgoing pod-to-pod traffic to node ip
+    # kubelet does not handle non-masquerade-cidr for containerd so we will have to handle it ourselves here
```

Comment:
main concern is  that we're hiding hard to debug rules additions into this fine.
In the same file we already have this 
https://github.com/Azure/AgentBaker/blob/1e0dd69d327a1f102ae6c913d1d660909f7b1500/parts/linux/cloud-init/nodecustomdata.yml#L483